README.md

```diff
@@ -146,3 +147,73 @@ prefixed with the attribute `#[verbose]` to get an idea of what they are doing.
   this seamingly mutual dependency using HB.
 
 </p></details>
+
+### Plan B
+
+Scared of making your project depend on HB? This section is for you.
+
+HB is based on a thick layer of software which we plan to maintain, but we
+also understand it can look scary. Hence this insurance plan. By passing
+the attribute `#[log]` each command prints Coq commands which are equivalent to
+its effect. By replacing each HB command by its equivalent Coq commands, you
+can eliminate the dependency on HB from your project.
+
+This is a "plan B", by looking at the output of`#[log]` you will realize that
+HB commands are much nicer (and shorter) than the equivalent Coq code. The
+point of a "plan B" is to avoid nightmares, not to be nicer than plan A ;-)
+
+How can you be ensure plan B works? We provide tools to check that, see
+the details below.
+
+<details><summary>(click to expand)</summary><p>
+
+
+Hierarchy Builder commands can log their equivalent vernacular commands
+to "patch" file (extension `.hb`). In order to do so, one has to
+compile the project with the `COQ_ELPI_ATTRIBUTES` variable set. Eg
+
+```shell
+COQ_ELPI_ATTRIBUTES='hb(log(raw))' make
+```
+
+The `hb` command line utility, provided by the `coq-hierarchy-builder` package,
```

Comment:
Maybe a more informative name would be `coq-hb` ?

Reply:
ok, I'll then call it `coq.hb` which will make it also work in the snap package with no other change.